hibernate-core/src/main/java/org/hibernate/hql/internal/ast/tree/FromElementType.java

```diff
@@ -368,6 +365,15 @@ public QueryableCollection getQueryableCollection() {
 		return queryableCollection;
 	}
 
+	public String getPropertyTableName(String propertyName) {
+		checkInitialized();
+		if (this.persister != null) {
```

Comment:
Same here about Checkstyle. I suppose we are not strict on the padding rule.

Reply:
fixed